src/objects/list/list.stories.mdx

```diff
@@ -12,8 +13,26 @@ const tagNames = ['ul', 'ol'];
 
 <Meta
   title="Objects/List"
+  parameters={{ docs: { inlineStories: false } }}
   argTypes={{
     class: { type: { name: 'string' } },
+    columns: {
+      control: {
+        type: 'range',
+        min: 2,
```

Comment:
Should `min` be `1`? Otherwise there isn't a way from the story controls to enable and then disable columns.